integration_tests/tests/apply/new.cy.ts

```diff
@@ -20,15 +22,17 @@ context('Risks', () => {
         oasysSections: riskData,
       })
     })
+    // and existing applications
+    cy.task('stubApplications', { applications })
```

Comment:
```suggestion
    cy.task('stubApplications', applications)
```

think we just pass in the list here rather than an object

Reply:
ah yes, and I think that fixes the issue you raised above. Thanks!